solrmarc_mixin/src/de/unituebingen/ub/ubtools/solrmarcMixin/TuelibMixin.java

```diff
@@ -726,23 +726,61 @@ public String getContainerVolume(final Record record) {
         return results;
     }
 
+    private static Map<String, String> GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP;
+    static {
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("verfasser", "aut");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("verfasserin", "aut");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("verleger", "hg");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("verlegerin", "hg");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("herausgeber", "hg");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("herausgeberin", "hg");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("übersetzer", "trl");
+        GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.put("übersetzerin", "trl");
+    }
+
+    private String translateAuthorRole(final String germanRole) {
+        if (GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.containsKey(germanRole))
```

Comment:
Das geht mit einem Lookup weniger:

    String translation = GERMAN_AUTHOR_ROLE_TO_ENGLISH_MAP.get(germanRole);
    if (translation == null) {
        System.err.println("No English mapping for \"" + germanRole + "\" found!");
    }